lib/flipper/adapters/active_record.rb

```diff
@@ -218,6 +236,10 @@ def set(feature, gate, thing, options = {})
         nil
       end
 
+      def delete(feature, gate)
+        @gate_class.where(feature_key: feature.key, key: gate.key).destroy_all
```

Comment:
```suggestion
        @gate_class.where(feature_key: feature.key, key: gate.key).delete_all
```
 Not that this is a hot code path, but any reason to destroy here vs delete?